projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -268,6 +268,7 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
     /** @hidden @internal */
     private setSelection(item: IgxDropDownItemBase) {
         if (item && item.value !== undefined && item.value !== null) {
+            this.navigateItem(item.index);
```

Comment:
It may be best if we use focus and blur handlers, in order to cover all focus scenarios and synch with the selected item like:             
(focus)="this.focusedItem = this.selectedItem"